pom.xml

```diff
@@ -59,8 +59,10 @@
         <url>http://github.com/killbill/killbill</url>
     </issueManagement>
     <properties>
+        <check.fail-enforcer>false</check.fail-enforcer>
```

Comment:
You can use parent pom `0.145.3-e8112a5-SNAPSHOT`.

Reply:
Fixed in https://github.com/killbill/killbill/pull/1695/commits/bb4bf4dd1168409ec025644a41307fb43aab2f85